index.js

```diff
@@ -1363,7 +1363,10 @@ function isBlock (index) {
 
 function defaultStorage (dir) {
   return function (name) {
-    return raf(name, {directory: dir})
+    try {
+      var lock = name === 'bitfield' ? require('fd-lock') : null
+    } catch (err) {}
```

Comment:
Perhaps a call to `debug()` might be helpful here, in case this ever needs to be debugged.